### PR TITLE
Describe peering

### PR DESCRIPTION
**Pull request by michielbdejong**

### CODE REVIEW

File: 0010-connector-to-connector-protocol/0010-connector-to-connector-protocol.md [hunk 1]
```diff
@@ -10,6 +10,31 @@ Connectors discover their peers through out-of-band communication, or by looking
...
+
+In WebFinger-based discovery, both peers look up each other's host resource, for instance:
+
+* the server wallet1.com looks up https://wallet2.com/.well-known/webfinger?resource=https://wallet2.com
+* the server wallet2.com looks up https://wallet1.com/.well-known/webfinger?resource=https://wallet1.com
+* out of band, they agree on a currency code and a currency scale for the peering ledger
+
+This way, each peer has the other peer's public key. They now use ECDH to create a shared secret, from which a ledger prefix and an auth token are derived.
+
+In `ilp_secret`-based peering, each peer generates an `ilp_secret` URN and gives that to the other one, to achieve the same:
```
Comment (emschwartz) [inline 0010-connector-to-connector-protocol/0010-connector-to-connector-protocol.md]: I'm not sure I would add this yet. @justmoon made a good point that the `ilp_secret` format is longer and doesn't necessarily provide any benefit over using the plain rpc connection string. We need to discuss whether this is the right format to go with.

**Comment by michielbdejong** [comment]
Maybe if ilp-kit will peer over CLP, can we get rid of the annoying peer ledger prefix stuff in the peering protocol - we can probably come up with a simpler peering protocol and describe that as we start using it.

See also https://github.com/interledgerjs/ilp-kit/issues/466

I'll revisit this after my holidays.

**Comment by michielbdejong** [comment]
@emschwartz bump

File: 0010-connector-to-connector-protocol/0010-connector-to-connector-protocol.md [hunk 2]
```diff
@@ -1,14 +1,72 @@
...
+draft: 3
 ---
 # Connector to Connector protocol
 
 ## Discovery
 
 Connectors discover their peers through out-of-band communication, or by looking at https://connector.land and contacting the administrator of another connector.
 
-Once peered, two connectors have a ledger between them; this is often a ledger with just two accounts, often administered collaboratively by the two connectors.
+Once peered, two connectors have a ledger between them; this is often a ledger with just two accounts.
```
Comment (emschwartz) [inline 0010-connector-to-connector-protocol/0010-connector-to-connector-protocol.md]: > this is often a ledger with just two accounts.

That raises more questions than it answers. What is a ledger with two accounts? Why is that the option that's often chosen?

File: 0010-connector-to-connector-protocol/0010-connector-to-connector-protocol.md [hunk 3]
```diff
@@ -1,14 +1,72 @@
...
+
+Whether discovery is done with or without use of WebFinger, each peer ends up knowing:
+
+* the protocol to use to send information like route broadcasts to the other connector
+* the protocol to use to add updates to the peering ledger
+* the currency code for the peering ledger
+* the currency scale for the peering ledger
+
+### WebFinger-based discovery
+In WebFinger-based discovery, both peers still need some out-of-band communication channel, over which one
```
Comment (emschwartz) [inline 0010-connector-to-connector-protocol/0010-connector-to-connector-protocol.md]: Maybe mention that this is normally something like talking in person or over email/chat

File: 0010-connector-to-connector-protocol/0010-connector-to-connector-protocol.md [hunk 4]
```diff
@@ -1,14 +1,72 @@
...
+* the server wallet2.com looks up https://wallet1.com/.well-known/webfinger?resource=https://wallet1.com
+
+This way, each peer has the other peer's public key. They now use ECDH to create a shared secret, from which a ledger prefix and an auth token are derived,
+as implemented in [ilp-kit](https://github.com/interledgerjs/ilp-kit).
+
+### Discovery without WebFinger
+When the two connectors do their discovery without WebFinger, the first peer tells the other:
+* their intent, "Let's peer without WebFinger!"
+* a BTP URI for the other connector to use
+* a BTP version to use (currently either 'BTP/alpha' or 'BTP/1.0')
```
Comment (emschwartz) [inline 0010-connector-to-connector-protocol/0010-connector-to-connector-protocol.md]: Probably worth mentioning that it's normally 1.0

File: 0010-connector-to-connector-protocol/0010-connector-to-connector-protocol.md (hunk 2 above)
Comment (michielbdejong) [reply 0010-connector-to-connector-protocol/0010-connector-to-connector-protocol.md]: ok, that was a phrase that was already there before this PR, but I removed it.

**Comment by michielbdejong** [comment]
@emschwartz .

**Pull request merged**